flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -54,7 +55,7 @@ class FlxPexParser
 		var angle = minMax("angle", config);
 		
 		var startSize = minMax("startParticleSize", config);
-		var finishSize = minMax("finishParticleSize", "FinishParticleSizeVariance", config);
+		var finishSize = minMax("finishParticleSize", "finishParticleSizeVariance", config);
```

Comment:
Sure. It is apparently a typo which originated in PD and was eventually fixed: http://forum.starling-framework.org/topic/this-might-be-a-typo-in-staring-extension-particle-system so best to support either version.